send-to-bluetooth@asgharalitaj/files/send-to-bluetooth@asgharalitaj/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+    "description": "Send a file to bluetooth",
+    "uuid": "send-to-bluetooth@asgharalitaj",
+    "name": "send to bluetooth",
+    "author": "Asghar Ali Taj",
```

Comment:
```suggestion
    "author": "AsgharAliTaj",
```

Reply:
i fixed the user name.

Reply:
send via bluetooth through out? even the folder names and stuff?

Reply:
different file fixed as well.

Reply:
my internet is dead beat so it takes some time..rofl